src/main/resources/jupyter/cluster-site.conf

```diff
@@ -9,12 +9,45 @@
     SSLVerifyClient require
     SSLVerifyDepth 10
 
+    RewriteEngine on
+
+    # RStudio
+
+    RewriteCond %{HTTP:Upgrade} =websocket
+    RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio/.* [NC]
+    RewriteRule /proxy/[^/]*/[^/]*/rstudio/(.*) ws://127.0.0.1:8001/$1  [P,L]
+
+    RewriteCond %{HTTP:Upgrade} !=websocket
+    RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio/.* [NC]
+    RewriteRule /proxy/[^/]*/[^/]*/rstudio/(.*) http://127.0.0.1:8001/$1 [P,L]
+
+    ProxyPass / http://127.0.0.1:8001/ retry=10
```

Comment:
I'm confused why we `ProxyPass /` here as well as below in the legacy Jupyter section. I would expect to have a more specific `ProxyPass` for RStudio, and do `/` in the catch-all. Something like:
```
# RStudio 

<rewrite rules>

ProxyPass /proxy/[^/]*/[^/]*/rstudio/ http://127.0.0.1:8001/
ProxyPassReverse /proxy/[^/]*/[^/]*/rstudio/ http://127.0.0.1:8001/

# Jupyter (legacy path)

<rewrite rules>

ProxyPass / http://127.0.0.1:8000/ retry=10
ProxyPassReverse / http://127.0.0.1:8000/
```
But if this works then maybe my understanding is incorrect..